packages/blade/src/components/Input/BaseInput/AnimatedBaseInputWrapper.web.tsx

```diff
@@ -152,6 +152,7 @@ to {
         }
       }}
       isTableInputCell={rest.isTableInputCell}
+      className="base-input-wrapper"
```

Comment:
Just once check if we're not doing `<AnimatedBaseInputWrapper className="something" />` or `styled(AnimatedBaseInputWrapper)` somewhere because this would override that